optuna/integration/skopt.py

```diff
@@ -46,7 +48,7 @@ def objective(trial):
 
                 sampler = optuna.integration.SkoptSampler()
```

Comment:
The `SkoptSampler` requires `scikit-optimize`. Please add it to the requirements of `doctest` in `setup.py`.